tsml_eval/experiments/experiments.py

```diff
@@ -110,6 +111,12 @@ def run_classification_experiment(
         Whether to generate train files or not. If true, it performs a 10-fold
         cross-validation on the train data and saves. If the classifier can produce its
         own estimates, those are used instead.
+    ignore_custom_train_estimate : bool, default=False
+        todo
```

Comment:
Is this just todo for later?

Reply:
yeah gonna leave these for another PR. the docs are generally a bit of a mess 🙂